### PR TITLE
Fix for 20791: Adds command to update the tools to the latest version…

### PR DESCRIPTION
**Pull request by riscie**
… to the version mismatch error message

### CODE REVIEW

**Comment by riscie** [comment]
Fix for #20792

File: src/EFCore.Design/Properties/DesignStrings.resx [hunk 1]
```diff
@@ -343,7 +343,7 @@ Change your target project to the migrations project by using the Package Manage
     <value>Using design-time services from assembly '{referencedAssembly}'.</value>
   </data>
   <data name="VersionMismatch" xml:space="preserve">
-    <value>The EF Core tools version '{toolsVersion}' is older than that of the runtime '{runtimeVersion}'. Update the tools for the latest features and bug fixes.</value>
+    <value>The EF Core tools version '{toolsVersion}' is older than that of the runtime '{runtimeVersion}'. Update the tools for the latest features and bug fixes. Use 'dotnet tool update --global dotnet-ef' to update to the latest version.</value>
```
Comment (smitpatel) [inline src/EFCore.Design/Properties/DesignStrings.resx:346]: This may not be accurate if the tool is installed locally.

Comment (riscie) [reply src/EFCore.Design/Properties/DesignStrings.resx:346]: I see. Splitting it up (global / local) would make the error message quite long. Do you still think the benefit is big enough to have it in there?

Comment (smitpatel) [reply src/EFCore.Design/Properties/DesignStrings.resx:346]: Having wrong command is worse than having no command.

Comment (riscie) [reply src/EFCore.Design/Properties/DesignStrings.resx:346]: Well sure. My question is wether we should include both commands.

Comment (ajcvickers) [reply src/EFCore.Design/Properties/DesignStrings.resx:346]: Seems like a link to conceptual documentation would be better here. Probably pointing to https://docs.microsoft.com/en-us/ef/core/miscellaneous/cli/dotnet

Comment (riscie) [reply src/EFCore.Design/Properties/DesignStrings.resx:346]: I agree a link seems better. But this one does not include the update command for tools.

Comment (ajcvickers) [reply src/EFCore.Design/Properties/DesignStrings.resx:346]: Yes, the documentation will need to be updated as well.

Comment (ntziolis) [reply src/EFCore.Design/Properties/DesignStrings.resx:346]: Please include the most common solution directly in the error message + link for more details. this is how angular cli etc does it and it seems to work well

Comment (riscie) [reply src/EFCore.Design/Properties/DesignStrings.resx:346]: Stumbled upon this today again. Any chance we can make progress on this (small) issue? Many people need to google on how to update tools. See this popular SO post: https://stackoverflow.com/questions/52108659/need-to-update-ef-core-tools

**Comment by janissimsons** [comment]
Why isn't this merged? Just add the most common command that can be used to fix the error. If the most common command doesn't work (in case of locally installed) then just add a link to documentation. 
Currently the most common way to fix this is:
1) Search the error message online
2) Find the first Stackoverflow link (https://stackoverflow.com/questions/52108659/need-to-update-ef-core-tools)
3) Copy and run the command: dotnet tool update --global dotnet-ef

Why not just put it in error message? You don't need to support every obscure scenario, in 99% cases people install the EF tools together with VS, not locally.

**Comment by riscie** [comment]
I don't understand either @janissimsons. There seems to be resistance because of edge-cases, that does more harm then good.

**Comment by riscie** [comment]
Thank you @ajcvickers

**Pull request merged**